src/View/Requirements_Backend.php

```diff
@@ -791,10 +839,24 @@ public function includeInHTML($content)
         }
 
         // Add all inline JavaScript *after* including external files they might rely on
-        foreach ($this->getCustomScripts() as $script) {
+        foreach ($this->getCustomScripts() as $key => $script) {
+            // Build html attributes
+            $curScriptAttributes = null;
+            if (array_key_exists($key, $this->customScriptAttributes)) {
+                $curScriptAttributes = $this->customScriptAttributes[$key];
+            }
+            $customHtmlAttributes = [
+                'type' => 'application/javascript'
+            ];
+            if (isset($curScriptAttributes)) {
+                $customHtmlAttributes['type'] = $curScriptAttributes['type'];
+                if (!empty($curScriptAttributes['crossorigin'])) {
+                    $customHtmlAttributes['crossorigin'] = $curScriptAttributes['crossorigin'];
+                }
+            }
```

Comment:
```suggestion
            // Build html attributes
            $customHtmlAttributes = ['type' => 'application/javascript'];
            foreach ($this->customScriptAttributes[$key] as $attrKey => $attrValue) {
                $customHtmlAttributes[$attrKey] = $attrValue;
            }
```
Fix issue where `type` attribute was required.  Also just general refactoring.